rb/lib/selenium/webdriver/safari.rb

```diff
@@ -24,6 +24,14 @@ module Selenium
   module WebDriver
     module Safari
       class << self
+        def technology_preview
+          "/Applications/Safari\ Technology\ Preview.app/Contents/MacOS/safaridriver"
+        end
+
+        def technology_preview=(value = true)
```

Comment:
You shouldn't be using default argument for setter, it should be used like this:

```ruby
Safari.technology_preview=
```

Maybe it's then better to use bang-method?